activesupport/test/concern_test.rb

```diff
@@ -54,6 +54,12 @@ module Foo
     include Bar, Baz
   end
 
+  module Qux
+    extend ActiveSupport::Concern
```

Comment:
No, I just need to manually call Object.extend in that case. Should I remove it and add Object.extend call?